### PR TITLE
Feature/use claim status

### PR DESCRIPTION
**Pull request by dwybourn**

### CODE REVIEW

File: api/src/test/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimantRepositoryTest.java [hunk 1]
```diff
@@ -68,76 +69,98 @@ void shouldHandleSqlInjectionAttempts() {
...
-        Boolean claimantExists = claimantRepository.eligibleClaimExistsForDwpHousehold(claimant.getDwpHouseholdIdentifier());
+        Boolean claimantExists = claimantRepository.liveClaimExistsForDwpHousehold(claimant.getDwpHouseholdIdentifier());
 
         //Then
         assertThat(claimantExists).isFalse();
     }
 
-    @Test
-    void shouldReturnClaimantDoesNotExistWhenDwpHouseholdIdentifierExistsAndStatusIsNotEligible() {
+    @ParameterizedTest(name = "Should return a live claim does not exists when dwp household exists and the claim status is {0}")
```
Comment (mikeSmith08) [inline api/src/test/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimantRepositoryTest.java]: Grammar check, you probably meant to say `Should return that a live claim does not exist...`

File: api/src/test/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimantRepositoryTest.java [hunk 2]
```diff
@@ -149,33 +172,44 @@ void shouldReturnClaimantDoesNotExistWhenHmrcHouseholdIdentifierDoesNotExist() {
...
-        Boolean claimantExists = claimantRepository.eligibleClaimExistsForHmrcHousehold(claimant.getHmrcHouseholdIdentifier());
+        Boolean claimantExists = claimantRepository.liveClaimExistsForHmrcHousehold(claimant.getHmrcHouseholdIdentifier());
 
         //Then
         assertThat(claimantExists).isFalse();
     }
 
-    @Test
-    void shouldReturnClaimantDoesNotExistWhenHmrcHouseholdIdentifierExistsAndStatusIsNotEligible() {
+    @ParameterizedTest(name = "Should return a live claim does not exists when hmrc household exists and the claim status is {0}")
```
Comment (mikeSmith08) [inline api/src/test/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimantRepositoryTest.java]: Same grammar problem here too.

File: api/src/test/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimantRepositoryTest.java [hunk 3]
```diff
@@ -149,33 +172,44 @@ void shouldReturnClaimantDoesNotExistWhenHmrcHouseholdIdentifierDoesNotExist() {
...
-        Boolean claimantExists = claimantRepository.eligibleClaimExistsForHmrcHousehold(claimant.getHmrcHouseholdIdentifier());
+        Boolean claimantExists = claimantRepository.liveClaimExistsForHmrcHousehold(claimant.getHmrcHouseholdIdentifier());
 
         //Then
         assertThat(claimantExists).isFalse();
     }
 
-    @Test
-    void shouldReturnClaimantExistsWhenHmrcHouseholdIdentifierExistsAndStatusIsEligible() {
+    @ParameterizedTest(name = "Should return a live claim does exists when hmrc household exists and the claim status is {0}")
```
Comment (mikeSmith08) [inline api/src/test/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimantRepositoryTest.java]: Ditto...

File: api/src/test/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimantRepositoryTest.java (hunk 1 above)
Comment (dwybourn) [reply api/src/test/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimantRepositoryTest.java]: Done.

File: api/src/test/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimantRepositoryTest.java (hunk 2 above)
Comment (dwybourn) [reply api/src/test/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimantRepositoryTest.java]: Done.

File: api/src/test/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimantRepositoryTest.java (hunk 3 above)
Comment (dwybourn) [reply api/src/test/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimantRepositoryTest.java]: Done.

**Pull request merged**